src/components/form/HdRange.vue

```diff
@@ -261,8 +261,8 @@ export default {
   $range: &;
   height: $sp-l;
   position: relative;
-  display: flex;
-  align-items: center;
+  display: flex !important;
+  align-items: center !important;
```

Comment:
This wasn't a scoped component, so it could affect projects overwriting these (although I doubt we would overwrite this).

Why did you have to force important here, if there are no other CSS changes in this pull request? Is it something coming from storybook?